opentelemetry-instrumentation/README.rst

```diff
@@ -22,7 +22,7 @@ This package provides a couple of commands that help automatically instruments a
     For more info about ``opentelemetry-distro`` check `here <https://opentelemetry-python.readthedocs.io/en/latest/examples/distro/README.html>`__
     ::
 
-        pip install opentelemetry-distro[otlp]
+        pip install opentelemetry-distro
```

Comment:
Looks like this is how it is installed (also specified in https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-distro#installation)

Reply:
That's different. The auto instrumentation requires it this way otherwise without any exporter users won't see any telemetry.